test/parallel/test-performance-measure.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+
+const { PerformanceObserver, performance } = require('perf_hooks');
+const DELAY = 1000;
+
+const expected = ['Start to Now', 'A to Now', 'A to B'];
+const obs = new PerformanceObserver(common.mustCall((items) => {
+  const entries = items.getEntries();
+  const { name, duration } = entries[0];
+  assert.ok(duration > DELAY);
+  assert.strictEqual(expected.shift(), name);
+  performance.clearMarks();
```

Comment:
IIUC, this API should not be called from observer callbacks, although the test passed. It's intended to be called on where performance marks are measured.